conf/messages

```diff
@@ -5,6 +5,8 @@ common.pageTitle = {0} - {1} - GOV.UK
 
 app.common.back = Back
 
+error.summary.title = There is a problem
```

Comment:
As part of having an error on the page - we need to modify the page title to prefix `Error: `
I've done this in https://github.com/hmrc/penalties-appeals-frontend/commit/42792af03390c359b9bb8f8f70cf34cd1dde5bd5